src/Actions/HandleCallbackAction.php

```
@@ -23,14 +24,12 @@ public function __construct(
 
     public function handle(CallbackPayload $payload): Transaction
     {
-        $transaction = $this->findOrCreateTransaction->handle($payload);
-
         if (! Sisp::validateCallback($payload)) {
```

PHPMD (2.15.0)

Avoid using static access to class '\Akira\Sisp\Facades\Sisp' in method 'handle'. (undefined)

(StaticAccess)